content/strategic/wage-war/plugin.js

```diff
@@ -80,7 +80,7 @@ const calcWarringPotential = (df) => (predator, defenseThreshold = 0) => (prey)
     }
     try {
         const sending = predator.energyCap * (parseInt(100 - defenseThreshold) / 100); // @todo: calculate predator energyCap as a factor of full energyCap at max level instead of predator.energyCap.
```

Comment:
Should sending still be a variable if its not being used anymore?

Whats the logic on the code change?

Reply:
it looks like sending is still used. it just got moved over.